Source/WebKit/Platform/ios/PlaybackSessionInterfaceLMK.mm

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
I don't think this file was actually moved from WebCore?

Reply:
Git is showing that it was deleted from WebCore and that a new file was created in WebKit. I'm not sure how to make it know that it was just moved- it seems like it worked for PlaybackSessionInterfaceLMK.h though.